src/grid.h

```diff
@@ -334,7 +334,7 @@ struct Grid {
     }
 
     Selection SelectAll() { return Selection(this, 0, 0, xs, ys); }
-    void ImageRefCount() { foreachcell(c) c->ImageRefCount(); }
+    void ImageRefCount(bool includefolded) { if(includefolded || !folded) foreachcell(c) c->ImageRefCount(includefolded); }

```

Comment:
good to spread more complex functions over multiple lines

Reply:
Ok I did this - hopefully right